prow/jobs/kyma/kyma-compass-integration.yaml

```diff
@@ -47,7 +47,8 @@ presubmits: # runs on PRs
               cpu: 80m
     branches:
       - ^master$
-    run_if_changed: "^(resources/compass|resources/compass-runtime-agent|installation)/"
```

Comment:
Shouldn't we launch this job similarly as `gke-upgrade` or other Kyma Integration jobs? It doesn't make sense to run it always:

https://github.com/kyma-project/test-infra/blob/master/prow/jobs/kyma/kyma-integration.yaml#L546